src/qt/bitcoingui.h

```diff
@@ -90,12 +90,14 @@ class BitcoinGUI : public QMainWindow
     QAction *historyAction;
     QAction *quitAction;
     QAction *sendCoinsAction;
+    QAction *sendCoinsMenuAction;
```

Comment:
createActions, which is part of the constructor, initialises them. Regardless, I added the (unnecessary) zeroing as requested.